projects/packages/import/src/endpoints/class-post.php

```diff
@@ -138,4 +141,63 @@ protected function extract_terms_ids( $request, $taxonomy ) {
 			return array();
 		}
 	}
+
+	/**
+	 * Processes the metadata of a WordPress post when creating or updating it.
+	 *
+	 * @param \WP_Post $post An object representing the post being created or updated.
+	 * @param mixed    $request An object containing the metadata being added to the post.
+	 * @param bool     $creating A flag indicating whether the post is being created or updated.
+	 * @return void
+	 */
+	public function process_post_meta( \WP_Post $post, $request, $creating ) {
+		$metas = $request->get_param( 'meta' );
+
+		if ( empty( $metas ) ) {
+			return;
+		}
+
+		$meta_keys_array = $this->filter_post_meta_keys( $metas );
+		// Adding it to the whitelist
+		Settings::update_settings( array( 'post_meta_whitelist' => $meta_keys_array ) );
+
+		if ( is_array( $metas ) ) {
+			foreach ( $metas as $meta_key => $meta_value ) {
+
+				$meta_value = maybe_unserialize( $meta_value );
+				if ( ! $creating && $meta_key === '_edit_last' ) {
```

Comment:
Specific meta values cannot be saved as-is but need some help wpcom side. For example, `_edit_last` has saved the ID of the user that modified the post. On wpcom we should do this:

1. A post meta that is known to have saved an ID of a user we should change it with the remapped user
2. A post meta that is known to have saved an ID of an external entity, this ID must be changed querying the DB like it happens in the [comment endpoint](https://github.com/Automattic/jetpack/blob/fix/meta-import/projects/packages/import/src/endpoints/class-comment.php#L67-L71)